.travis.yml

```diff
@@ -50,7 +50,7 @@ matrix:
       rust: nightly
       os: linux
       script:
-        - cargo doc --features docs
+        - cargo doc --features docs unstable
```

Comment:
This is good. Could you also add the following item to the main `script` section at the bottom of this file?

```
  - cargo test --all --docs
```

It's to make sure that all doc tests pass with stable `async-std`, but examples and integration tests (which might use unstable features) pass with unstable `async-std`.

Reply:
done!